base/util_testing.go

```diff
@@ -464,49 +466,45 @@ func SetUpGlobalTestLogging(m *testing.M) (teardownFn func()) {
 	return func() { GlobalTestLoggingSet.Set(false) }
 }
 
-// SetUpTestLogging will set the given log level and log keys,
-// and return a function that can be deferred for teardown.
-//
-// This function will panic if called multiple times without running the teardownFn.
-//
-// To set multiple log keys, append as variadic arguments
-// E.g. KeyCache,KeyDCP,KeySync
-//
-// Usage:
-//     teardownFn := SetUpTestLogging(LevelDebug, KeyCache,KeyDCP,KeySync)
-//     defer teardownFn()
+// canCleanup is a subset of testing.TB used for testing the test logging.
+type canCleanup interface {
+	Cleanup(func())
+}
```

Comment:
With the mock comment above addressed, I think the functions can just take a `testing.TB` interface instead.